resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/DeTwineStrings.kt

```diff
@@ -0,0 +1,149 @@
+        editFeedName = "Bearbeiten",
+        errorUnsupportedFeed = "Der Link enthält keinen RSS/Atom-Feed.",
+        errorMalformedXml = "Der bereitgestellte Link enthält keinen gültigen RSS/Atom-Feed",
+        errorRequestTimeout =
+        "Zeitüberschreitung, überprüfen Sie Ihre Netzwerkverbindung und versuchen Sie es später erneut",
+        errorFeedNotFound = { "($it): Unter dem angegebenen Link wurde kein Inhalt gefunden." },
+        errorServer = {
+            "($it): Serverfehler. Bitte versuchen Sie es später noch einmal oder wenden Sie sich an den Website-Administrator."
+        },
+        errorTooManyRedirects =
+        "Die angegebene URL enthält zu viele Weiterleitungen. Bitte verwenden Sie eine andere URL.",
+        errorUnAuthorized = {
+            "($it): Sie sind nicht berechtigt, auf Inhalte unter diesem Link zuzugreifen."
+        },
+        errorUnknownHttpStatus = { "Inhalt konnte nicht mit HTTP-Code geladen werden: ($it)" },
+        postsSearchHint = "Beiträge durchsuchen",
+        searchSortNewest = "Neueste",
+        searchSortNewestFirst = "Das neuste zuerst",
+        searchSortOldest = "Älteste",
+        searchSortOldestFirst = "Die ältesten zu erst",
+        searchResultsCount = { count ->
+            when (count) {
+                1 -> "$count Ergebnis"
+                else -> "$count Ergebnisse"
+            }
+        },
+        bookmark = "Lesezeichen",
+        bookmarks = "Lesezeichen",
+        bookmarksPlaceholder = "Mit Lesezeichen versehene Beiträge werden hier angezeigt 🔖",
+        settings = "Einstellungen",
+        moreMenuOptions = "Weitere Menüoptionen",
+        settingsHeaderBehaviour = "Verhalten",
+        settingsHeaderFeedback = "Feedback und Fehlerberichte",
+        settingsHeaderOpml = "OPML",
+        settingsBrowserTypeTitle = "Verwenden Sie den In-App-Browser",
+        settingsBrowserTypeSubtitle =
+        "Wenn diese Option deaktiviert ist, werden Links in Ihrem Standardbrowser geöffnet.",
+        settingsEnableBlurTitle = "Aktivieren Sie die Unschärfe auf der Startseite",
+        settingsEnableBlurSubtitle =
+        "Wenn deaktiviert, werden Farbverläufe statt unscharfer Bilder angezeigt. Kann die Leistung verbessern.",
+        settingsShowUnreadCountTitle = "Anzahl der ungelesenen Beiträge anzeigen",
+        settingsShowUnreadCountSubtitle =
+        "Wenn diese Option deaktiviert ist, wird die Anzahl der ungelesenen Beiträge nicht im Feed-Blatt angezeigt",
+        settingsReportIssue = "Ein Problem melden",
+        settingsVersion = { versionName, versionCode -> "$versionName ($versionCode)" },
+        settingsAboutTitle = "Um Twine",
+        settingsAboutSubtitle = "Lernen Sie die Autoren kennen",
+        settingsOpmlImport = "Importieren",
+        settingsOpmlExport = "Export",
+        settingsOpmlImporting = { progress -> "Importieren.. $progress%" },
+        settingsOpmlExporting = { progress -> "Exportieren.. $progress%" },
```

Comment:
In Android we would usually use the 3 dots character. Not sure how it is down in a KMM project & your English text also does just contain two normal dots.

Reply:
the english counterpart is two .